test/gtest/ucm/malloc_hook.cc

```diff
@@ -132,6 +132,39 @@ class malloc_hook : public ucs::test {
             UCM_BISTRO_EPILOGUE;
             return res;
         }
+
+        static int madvise(void *addr, size_t length, int advise)
```

Comment:
maybe add also a non-bistro test? for example add madvise(DONTNEED) to `void test_thread::test()` - to test also reloc mode

Reply:
added to mmap_hooks::test